lib/chef/knife/cookbook_site_download.rb

```diff
@@ -29,6 +29,12 @@ class CookbookSiteDownload < Knife::SupermarketDownload
 
       banner "knife cookbook site download COOKBOOK [VERSION] (options)"
       category "deprecated"
```

Comment:
does category "deprecated" not do anything useful like this already?

Reply:
Nope

Reply:
if I thought we were going to do this more I'd add a deprecated method to the plugin that would handle it, but I think we're good until we look at migrating knife commands over to chef.